presto-docs/src/main/sphinx/connector/mysql.rst

```diff
@@ -215,16 +215,131 @@ The connector maps PrestoDB types to the corresponding MySQL types:
 
 No other types are supported.
 
+MySQL Connector Support
+=======================
+
+The MySQL connector allows querying and creating MySQL tables and schemas. Here are some examples of the SQL operations supported:
+
+CREATE SCHEMA
+-------------
+
+Create a new MySQL schema named `web`:
+
+.. code-block:: sql
+
+    CREATE SCHEMA mysql.web;
+
+.. code-block:: none
+
+    This connector does not support creating schemas
+
+CREATE TABLE
+------------
+
+Create a new MySQL table named `page_views` in the `web` schema:
+
+.. code-block:: sql
+
+    CREATE TABLE mysql.web.page_views (
+      user_id bigint,
+      page_url varchar,
+      ds date,
+      country varchar
+    );
+
+CREATE VIEW
+-----------
+
+Create a view named `view_page_views` for the `mysql.web.page_views` table:
+
+.. code-block:: sql
+
+    CREATE VIEW mysql.web.view_page_views AS SELECT user_id, country FROM mysql.web.page_views;
+
+.. code-block:: none
+
+    This connector does not support creating views
+
+
+INSERT INTO
+-----------
+
+Insert data into the `page_views` table:
+
+.. code-block:: sql
+
+    INSERT INTO mysql.web.page_views VALUES(1, 'https://example.com', current_date, 'country');
+
+CREATE TABLE AS SELECT
+----------------------
+
+Create a new table `page_views_new` from an existing table `page_views`:
+
+.. code-block:: sql
+
+    CREATE TABLE mysql.web.page_views_new AS SELECT * FROM mysql.web.page_views;
+
+SELECT
+------
+
+SELECT table operations are supported:
+
+.. code-block:: sql
+
+    SELECT * FROM mysql.web.page_views;
+
+ALTER TABLE
+-----------
+
+Alter table operations are supported:
+
+.. code-block:: sql
+
+    ALTER TABLE mysql.web.page_views ADD COLUMN zipcode VARCHAR;
+    ALTER TABLE mysql.web.page_views RENAME COLUMN zipcode TO location;
+    ALTER TABLE mysql.web.page_views DROP COLUMN location;
+
+TRUNCATE
+--------
+
+The MySQL connector can delete all of the data from tables without dropping the table:
+
+.. code-block:: sql
+
+    TRUNCATE TABLE mysql.web.page_views;
+
+DROP TABLE
+----------
+
+Drop the table `page_views`:
+
+.. code-block:: sql
+
+    DROP TABLE mysql.web.page_views;
+
+.. code-block:: none
+
+    DROP TABLE is disabled in this catalog
+
 MySQL Connector Limitations
----------------------------
-
-The following SQL statements are not yet supported:
-
-* :doc:`/sql/delete`
-* :doc:`/sql/alter-table`
-* :doc:`/sql/create-table` (:doc:`/sql/create-table-as` is supported)
-* :doc:`/sql/grant`
-* :doc:`/sql/revoke`
-* :doc:`/sql/show-grants`
-* :doc:`/sql/show-roles`
-* :doc:`/sql/show-role-grants`
+===========================
+
+The following SQL statements are not supported:
+
+- DELETE
```

Comment:
I liked the previous version as it included links.  Can you add links to these docs?

Reply:
This has been added.